Small update

ItzxDwi opened the pull request.

ItzxDwi committed:
Update EventListener.php

ItzxDwi committed:
Update VanishV2Task.php

ItzxDwi committed:
Update VanishV2.php

ItzxDwi committed:
Update VanishV2.php

ItzxDwi committed:
Update config.yml

sqrrrrl committed:
Tell don't ask principle and add xuid to PlayerListPacket

ItzxDwi committed:
Merge branch 'sqrrrrl:PM5' into PM5

ItzxDwi committed:
Update VanishV2.php

ItzxDwi committed:
Update VanishV2.php

ItzxDwi committed:
Update VanishV2.php

ItzxDwi committed:
Merge branch 'sqrrrrl:PM5' into PM5

ItzxDwi commented:
@sqrrrrl

sqrrrrl commented:
We actually can just remove the parts playing with the effects since hidePlayer() already hides the effects particles.

sqrrrrl committed:
Removed useless code

Removed the part hiding and revealing the effects particles as they are not needed.
